src/strategies/ethtrader-gov-weight/index.ts

```diff
@@ -0,0 +1,25 @@
+import fetch from 'cross-fetch';
+
+export const author = 'kohrts';
+export const version = '0.1.0';
+
+export async function strategy(space, network, provider, addresses, options, snapshot) {
+  const ETHTRADER_USERS_URL = 'https://raw.githubusercontent.com/EthTrader/donut.distribution/main/docs/users.json'
```

Comment:
Hi, @kohrts  like this owner of this repo can change the data any time and won't give us consistent results, We need this data from on-chain / IPFS so it won't change in the future, Can you think of any alternate way to get data based on block and network?

Reply:
Yes, all this data does originate on-chain and IPFS, but it requires more than 5 calls to compile it all.

Reply:
Is it 5 calls in total or 5 calls per address? If it's in total that should work within the strategy. If you use external service to get voting power you need to take the "snapshot" in consideration and store voting power history in a way that it always answer with same power based on the snapshot

Reply:
thanks, i get it! that makes a lot of sense. i'll make those updates